packages/react-component-library/src/components/Checkbox/Checkbox.stories.tsx

```diff
@@ -25,6 +25,7 @@ const MultipleItemsTemplate: ComponentStory<typeof Checkbox> = (props) => {
   function getProps(i: number): CheckboxProps {
     return {
       ...props,
+      variant: CHECKBOX_RADIO_VARIANT.NO_CONTAINER,
```

Comment:
This was breaking when opening the canvas in a new window (because Storybook was deciding to omit the `variant` arg for some reason), so moved this here to work around that.